tests/moduletool/test_module_tool_basics.py

```diff
@@ -167,7 +174,10 @@ def metadata_contents(version: str) -> str:
 
   cookiecutter~=1.7.0
   cryptography>1.0,<3.5
-            """.strip()
+[egg_info]
+tag_build = {version_tag}
+            """
```

Comment:
I removed the strip() here, because otherwise the whitespace between the `=` and `{version_tag}` gets removed when `version_tag` is an empty string. The trip in the if-block is redundant, so I removed it there too for consistency.